packages/mobile/src/components/core/TextInput.tsx

```diff
@@ -67,13 +68,12 @@ const useStyles = makeStyles(({ typography, palette, spacing }) => ({
     color: palette.neutral,
     fontFamily: typography.fontByWeight.medium,
     minWidth: 40,
+    flexGrow: 1,
     // Needed for android
     padding: 0
   },
   icon: {
-    fill: palette.neutralLight5,
-    height: spacing(4),
```

Comment:
worth noting the default size of these icons is 24x24 and we're removing this which would resize them to 16x16 - might want to check other usages?

Reply:
nice good call. there are other uses which rely on 16. gonna add icon style as a prop.